tdrs-frontend/nginx/buildpack.nginx.conf

```diff
@@ -20,8 +20,33 @@ http {
     server {
         listen {{port}};
 
-        root public;
-        index index.html index.htm;
-        try_files $uri $uri/ /index.html;
+        location = /edit-profile {
```

Comment:
Would there be any issues with combining these two nginx configurations into one file?

Having to maintain these URLs in both of these files along with the React Router code seems like overkill.

Reply:
We could have this file include the other file inside of the server directive I think, the buildpacks have to declare a server, but the docker config seems to be included under a server block from inside the container.